### PR TITLE
News-site-project Estefanía week 2 Technigo bootcamp

### PR DESCRIPTION
**Pull request by esteficodes**
This is a news site inspired in trap music performed by female artists. 

I completed the Blue level of this project, although I used grid to align my news cards, as I got better results with this. I struggled to make the website display 4 cards in desktop, 2 in tablet and 1 for mobile. I found this especially challenging as it was my first responsive website ever.

I will appreciate feedback regarding all the areas in which this project could be better, as I know I still have a lot to learn.

### CODE REVIEW

**Comment by HIPPIEKICK** [comment]
Hi Estefanía! Did you deploy your site to Netlify and can provide a link? :)

**Comment by esteficodes** [comment]
> Hi Estefanía! Did you deploy your site to Netlify and can provide a link? :)

Yes, I put it on Canvas, when I delivered the project. Here it goes again: https://wonderful-goodall-5c6d86.netlify.app/

File: code/index.html [hunk 1]
```diff
@@ -1,22 +1,129 @@
 <!DOCTYPE html>
-<html>
+<html lang="en">
   <head>
     <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
     <title>Project 2: The News Site</title>
```
Comment (HIPPIEKICK) [inline code/index.html:7]: You can change this to something personal!

File: code/index.html [hunk 2]
```diff
@@ -1,22 +1,129 @@
...
+ 
+  <!-- HEADER -->
+  
     <header>
-      <!-- Some kind of logo and navbar -->
+      <div class="hamburger">
+        <div></div>
+        <div></div>
+        <div></div>
+      </div>
```
Comment (HIPPIEKICK) [inline code/index.html:19]: Nice with a hamburger!

File: code/index.html [hunk 3]
```diff
@@ -1,22 +1,129 @@
...
+      <img src="images/Queens.png" alt="trap queens title">
+      <nav>
+        <a href="#">HOME</a>
+        <a href="#">NEWS</a>
+        <a href="#">ABOUT</a>
+      </nav>
```
Comment (HIPPIEKICK) [inline code/index.html:25]: Nice and clean 💯

File: code/index.html [hunk 4]
```diff
@@ -1,22 +1,129 @@
...
+      <div>
+        <h2>Smashing news!</h2>
+        <p>
+          What has Tina Trapper been up to? 
+          Find out everything about it. Lorem ipsum dolor sit amet consectetur adipisicing elit. Aliquam quo nihil voluptatem fuga et obcaecati rerum repellat enim repudiandae, vitae numquam
+          Consequuntur et minus esse reiciendis vero quia cumque molestiae?
+        </p>
+      </div>
+    </section>
     </section>
```
Comment (HIPPIEKICK) [inline code/index.html:42]: Nested elements should be indented one level, take a look at the [code guidelines](https://www.notion.so/technigo/Guidelines-for-how-to-write-good-code-69ffb59eac4e4358a7c5752efb873c4e) if you want to recap :)

File: code/index.html [hunk 5]
```diff
@@ -1,22 +1,129 @@
...
-  </body>
-</html>
+
+    <h3>What's hot? find it out here!</h3>
+
+     <!-- OTHER NEWS -->
+  <section class="news-container">
```
Comment (HIPPIEKICK) [inline code/index.html:47]: Siblings should be on the same level of indentation

File: code/index.html [hunk 6]
```diff
@@ -1,22 +1,129 @@
...
+  <p>Find out more here</p>
+   </div>
+   
+</section>
+   
+ <!-- FOOTER -->
+
+  <div class="footer">
+    <p>Developed by @esteficodes- A Technigo bootcamp project</p>
+  </div>
```
Comment (HIPPIEKICK) [inline code/index.html:126]: This could be a `<footer>` 🤩

File: code/style.css [hunk 7]
```diff
@@ -1,4 +1,206 @@
...
+}
+
+/*** OTHER NEWS ***/
+
+.news-container {
+  display:grid;
+  grid-column-gap: 16px;
+  grid-template-rows: repeat(8, 1fr);
+  grid-gap: 5px;  
+}
```
Comment (HIPPIEKICK) [inline code/style.css:124]: So nice that you tried out CSS grid! Looking good!

File: code/style.css [hunk 8]
```diff
@@ -1,4 +1,206 @@
...
+  .news-container {
+   grid-template-rows: repeat(2, 1fr);
+   grid-template-columns: repeat(4, 1fr);
+
+  }
+  .news-item {
+    width:100%;
+    height: fit-content;
+  }
 }
```
Comment (HIPPIEKICK) [inline code/style.css:191]: Looks a bit weird here, I think you can't have both 100% width on the image _and_ flex-direction: row; on the container:

Skärmavbild 2021-02-02 kl  11 11 41

**Review by HIPPIEKICK — CHANGES_REQUESTED** [review]
Looking good Estefanía! Look at my comment about indentation and give your HTML a second glance and you're good to go. 

We’ve also put together some general feedback that applies to many projects this first week. Check it out and take the tips with you to upcoming projects! 👉 [General Feedback - week 2](https://www.notion.so/technigo/Week-2-Project-news-site-bbe695289d154ff98d6a247cef873f07)

File: code/style.css (hunk 8 above)
Comment (esteficodes) [reply code/style.css:191]: I tried your suggestion but it still looks kind of more on the right side. I don't know why that is.

**Comment by esteficodes** [comment]
Hi @HIPPIEKICK I'm done with the requested changes. 😀

**Pull request closed without merging**